### PR TITLE
Alg2 split

### PR DESCRIPTION
**Pull request by flannery-denny**
The work on this branch is far from done, but we decided that we should probably merge it anyway, since the work on scatter plots and fitting models impacts that Data Science pathway and the unfinished stuff is all Algebra 2 related and will be completed before anything goes live on the web.

### CODE REVIEW

File: lessons/Data-Science/scatter-plots/langs/en-us/index.adoc [hunk 1]
```diff
@@ -48,85 +47,117 @@
...
 
 @QandA{
 @Q{Do you think that younger animals get adopted faster? Why or why not?}
 @A{The goal here is to have an open discussion and draw students in. Allow students to share their opinions freely. (For example: Yes, baby animals get adopted quickly because they're so cute! No, animals require too much work when they are young.)}
-@Q{What kind of data is `age`? What kind of data is `weeks`?}
+@Q{What kind of data is `age`? What kind of data is `weeks` to adoption?}
 @A{Both `age` and `weeks` are quantitative.}
 @Q{What kind of display would help us analyze the relationship between age and adoption time?}
 @A{Again, solicit students ideas and discuss why each display type would or would not work.}
+@A{@ifproglang{pyret}{Pie and }Bar charts won't work, because they show the _frequency_ of values in a _categorical_ column.}
```
Comment (schanzer) [inline lessons/Data-Science/scatter-plots/langs/en-us/index.adoc:64]: Nice proglang-specific catch!

File: lessons/Data-Science/scatter-plots/langs/en-us/index.adoc [hunk 2]
```diff
@@ -48,85 +47,117 @@
...
 
 @slidebreak
 
-@ifproglang{pyret}{Pie}@ifproglang{codap}{Dot plots} and Bar charts help us see the _frequency_ of values in a _categorical_ column. There are other visualizations, like histograms and box plots, that help us explore the _distribution_ of values in a _quantitative_ column.
-
-What we really want is a display that will help us search for *a relationship between two quantitative columns*, and that's exactly what scatter plots do.
-
 @lesson-point{
-Scatter plots reveal the potential relationship between two columns by plotting one on the x-axis and the other on the y-axis.
+Scatter plots are special because they help us visualize the relationship between two columns of data.
```
Comment (schanzer) [inline lessons/Data-Science/scatter-plots/langs/en-us/index.adoc]: the _potential_ relationship!

File: lessons/Data-Science/scatter-plots/langs/en-us/pages/creating-scatterplot.adoc
Comment (schanzer) [inline lessons/Data-Science/scatter-plots/langs/en-us/pages/creating-scatterplot.adoc:1]: I really like improvements to this lesson!

File: shared/langs/en-us/learningObjectives/alg2.json
Comment (schanzer) [inline shared/langs/en-us/learningObjectives/alg2.json:1]: Isn't the algebra.json file already really huge? Not sure why you want to include quadratics, exponentials, etc there. Maybe just rename this `non-linear.json`?

File: lessons/Data-Science/fitting-models/langs/en-us/pages/how-could-we-measure-good-fit-lizard.adoc [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+
+@fitb{}{}
+
+== Comparing Models
+
+@n Some other students drew the two lines below. Do you think `f(x)` or `g(x)` is a better model for this data? Why?
+
+[cols="1a,1a", frame="none", grid="none", stripes="none"]
+|===
+| @centered-image{../images/which-model.png}
```
[the remark this answers is not in the document]
Comment (flannery-denny) [reply lessons/Data-Science/fitting-models/langs/en-us/pages/how-could-we-measure-good-fit-lizard.adoc]: If I made a brand new image. We voted against that when we talked about changing the function names. Does it feel worth the time now?

File: lessons/Data-Science/fitting-models/langs/en-us/index.adoc [hunk 4]
```diff
@@ -0,0 +1,265 @@
...
+  ** Finally, on line 23 there's a line of code that places the outputs of the two `fit-model` expressions beside each other.
+- Click "Run" to test out how well the models fit the dataset.
+}
+
+@teacher{
+When students click "Run", the Interactive charts will open in the same order they are defined in the file. The first data visualization that pops open when we click "Run" will use `f`. Students will need to close this window in order to see the data visualization that uses `g`. And once they close that, they will see a thumbnail image in the Interactions Area of the two charts printed side by side.
+}
+
+@lesson-instruction{
+- Close the two interactive charts that pop up and then click on the thumbnail image in the Interactions Area to see it full-size.
```
[the remark this answers is not in the document]
Comment (flannery-denny) [reply lessons/Data-Science/fitting-models/langs/en-us/index.adoc:150]: They're being asked to notice and wonder here and it is pointed out to teachers in the part of the lesson that scaffolds comparing the two visualizations.

File: shared/langs/en-us/learningObjectives/alg2.json
Comment (flannery-denny) [reply shared/langs/en-us/learningObjectives/alg2.json:1]: @schanzer i can't see what this refers to. Trusting you resolved it.

**Comment by flannery-denny** [comment]
@schanzer Once we decide whether to live with or remake that fit-model image and how to resolve the starter file this will be ready to merge :)

**Pull request merged**